packages/web-sockets/package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "@booking-services/web-sockets",
+  "version": "0.0.1",
+  "description": "Configuration to our communication using websockets",
+  "keywords": [
+    "websockets"
+  ],
+  "author": "Hector Esau Macias Javier <hecttoressau@gmail.com>",
+  "homepage": "https://github.com/Platzi-Master-C9/backend-services/tree/main/packages/web-sockets#readme",
+  "license": "ISC",
+  "main": "src/index.js",
+  "directories": {
+    "lib": "src",
+    "test": "__tests__"
+  },
+  "files": [
+    "src"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hecttoressau/booking-services.git"
+  },
+  "scripts": {
+    "test": "jest"
+  },
```

Can you add scripts here for `lint` and `lint:fix`?
```suggestion
    "lint": "eslint .",
    "lint:fix": "eslint . --fix"
  },
```